content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md

```diff
@@ -13,64 +13,75 @@ further_reading:
-    
+5. Name it (e.g **DatadogSSO_test**).
+
+6. Click on **Add**.
+
+7. Once the your application is successfully added, go in **Configure single sign-on (required)**.
+
+8. Select the **SAML-based Sign-on** as Single Sign-on Mode.
+
+    {{< img src="account_management/faq/saml_based_sign_on.png" alt="Saml Based Sign on" responsive="true" popup="true" style="width:70%;">}}
+
+9. Navigate to the [Datadog SAML page][1], find the **Service Provider Entity ID** & **Assertion Consumer Service** URL on the right hand of the page.  Copy and paste those values in the **Identifier** and **Reply URL** text forms respectively:
+    In the datadog application:
+
+    {{< img src="account_management/faq/Step10Redo.png" alt="Step10Redo" responsive="true" popup="true" style="width:70%;">}}
 
-7. Select "Custom" -> "Add an unlisted application my organization is using" -> Enter a name like "DatadogSSOApp" as the application name. Then press the check button when complete
-    **NOTE**: if you don't see the "Add an unlisted application my organization is using" option, it means that you don't have the Premium subscription for Azure AD. Upgrade to Azure AD Premium and then refresh the page
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure2.png" alt="Active_Directory_-_Microsoft_Azure2" responsive="true" popup="true">}}
+    In the Azure portal:
 
-8. Once the Application has been created, select "Configure single sign-on"
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure3.png" alt="Active_Directory_-_Microsoft_Azure3" responsive="true" popup="true">}}
+    {{< img src="account_management/faq/set_values_azure.png" alt="Set value azure" responsive="true" popup="true" style="width:70%;">}}
 
-9. Select "Microsoft Azure AD Single Sign-On"
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure4.png" alt="Active_Directory_-_Microsoft_Azure4" responsive="true" popup="true">}}
+11. Set `user.mail` as **User Identifier** Value:
 
-10. Navigate to https://app.datadoghq.com/saml/saml_setup, find the Service Provider Entity ID & Assertion Consumer Service URL on the right hand of the page.  Copy and paste those values in the "Identifier" and "Reply URL" text forms respectively
-    {{< img src="account_management/faq/Step10Redo.png" alt="Step10Redo" responsive="true" popup="true">}}
-    {{< img src="account_management/faq/Step10Redo2.png" alt="Step10Redo2" responsive="true" popup="true">}}
+    {{< img src="account_management/faq/user_identifier.png" alt="User Identifier" responsive="true" popup="true" style="width:70%;">}}
 
-11. On the next page Download Metadata (XML), check the box confirming you have configured SSO and press next. Use this file in step 17
+12. Enter your **Notification Email** at the bottom of the page for when the active signing certificate approaches its expiration date, notifications are then sent to this email address with instructions on how to update the certificate:
```

Comment:
12. Enter your **Notification Email** at the bottom of the page. When the active signing certificate approaches its expiration date, notifications are sent to this email address with instructions on how to update the certificate: